tools/docker/docker-compile.pl

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env perl
+
+# copyright: Maciej Pasternacki
+# http://3ofcoins.net/2013/09/22/flat-docker-images/
+#
+# TODO: replace with something homemade
+
```

Comment:
He added his copyright but published it without a license (which means it's in the public domain). That entitles us to relicense it under the Boost license, I don't see a problem with that.